charts/daemonset-app/templates/sealedsecret.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
 spec:
   encryptedData:
     {{- range $key, $value := .Values.secrets }}
-    {{ $key | upper }}: {{ $value | quote }}
+    {{ $key }}: {{ $value | quote }}
```

Comment:
So .. I'm a bit confused. Why do we think this is a breaking change exactly? Are people using lower-cased environment variables?

Reply:
It's changing the behavior of the underlying template (people _may_ have been relying on this behavior).

That's why I thought not doing the same change for the existing `secrets.yaml` and  `kmssecrets.yaml`... but I'd like the added consistency and maybe we can make it part of this change (and do a maj version bump)?

Reply:
Ok. I don't feel too strongly - but I feel like the underlying  chart behavior should change and we should jsut consider that a minor version bump. It's reasonable that environment variables should be upper-cased..

Reply:
Great! My heart wanted to make it this behavior for everyone :) (but didn't want to step on any previous decisions that were made regarding that). Updated (and glad we did)!